ext/standard/array.c

```diff
@@ -6620,23 +6620,32 @@ static enum php_array_find_result php_array_find(const HashTable *array, zend_fc
 		if (!str_key) {
 			ZVAL_LONG(&args[1], num_key);
 		} else {
+			ZEND_ASSUME(!HT_IS_PACKED(array));
```

Comment:
This could benefit from a short explanatory comment.

Reply:
Ack. It's also not clear to me how this improves performance.